### DISCUSSION TITLE
Infura has no "projects" anymore, now it has "keys"

### DISCUSSION THREAD

**Discussion by fantauzzi** [smartcontractkit/full-blockchain-solidity-course-py#1605] [General]
Infura now call "keys" what it used to call "projects". You won't find a `Project ID` anymore in Infura's  `SETTINGS` tab; that is now called an `API KEY`, and is under the `ENDPOINTS` tab. So when you deploy your contracts via Infura, you should set the `WEB3_INFURA_PROJECT_ID` environment variable to the `API KEY` (in your `.env` file typically).

**Comment by cromewar** [comment]
Nice to know @fantauzzi 
Thanks for helping the community with this information!
